### PR TITLE
Sync various Date related files

### PR DESCRIPTION
**Pull request by victor-prdh**
Hi everyone,
Some work on the date section
Thanks for your review
cheers

### CODE REVIEW

**Comment by victor-prdh** [comment]
sorry, didn't have time to translate `reference/datetime/datetimeinterface/format.xml`, i just fix the link for now

File: reference/datetime/datetime/construct.xml [hunk 1]
```diff
@@ -9,16 +9,17 @@
...
-   Comme <methodname>DateTimeImmutable::__construct</methodname> mais
-   fonctionne avec <classname>DateTime</classname>. Considérer l'utilisation de
-   <classname>DateTimeImmutable</classname> et ses fonctionnalités à la place.
-  </para>
+  <para>&style.procedural;</para>
+  <methodsynopsis>
+   <type class="union"><type>DateTimeImmutable</type><type>false</type></type><methodname>date_create_immutable</methodname>
+   <methodparam choice="opt"><type>string</type><parameter>datetime</parameter><initializer>"now"</initializer></methodparam>
+   <methodparam choice="opt"><type class="union"><type>DateTimeZone</type><type>null</type></type><parameter>timezone</parameter><initializer>&null;</initializer></methodparam>
+  </methodsynopsis>
```
Comment (Girgias) [inline reference/datetime/datetime/construct.xml:22]: Wrong class as this should be about ``DateTime`` and not ``DateTimeImmutable``

Actual diff: http://doc.php.net/revcheck.php?p=plain&lang=fr&hbp=eeeb152d8a3f41463148e87e0c048c4fb168f292&f=reference/datetime/datetime/construct.xml&c=on

**Review by Girgias — CHANGES_REQUESTED** [review]
Other the one issue looks good

File: reference/datetime/datetime/construct.xml (hunk 1 above)
Comment (victor-prdh) [reply reference/datetime/datetime/construct.xml]: Oh yes my bad ! it's now fixed

File: reference/datetime/datetime/construct.xml [hunk 2]
```diff
@@ -9,16 +9,17 @@
...
```
Comment (Girgias) [inline reference/datetime/datetime/construct.xml:22]: Again please revert this as this is *not* how the English page is

**Review by Girgias — CHANGES_REQUESTED** [review]

File: reference/datetime/datetime/construct.xml (hunk 2 above)
Comment (victor-prdh) [reply reference/datetime/datetime/construct.xml]: omg, just understand what did i do... sorry i don't know from where i translated this

**Review by Girgias — APPROVED** [review]

**Comment by Girgias** [comment]
Thank you!

**Pull request merged**